src/algorithm/string.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+namespace TW {
+
+static std::vector<std::string> split(const std::string& input, char delimiter) {
```

Comment:
Is `split` in general `TW` namespace specific enough for string split? Maybe `stringsplit` or `ssplit`?